reactive/hacluster.py

```diff
@@ -0,0 +1,79 @@
+from charms.reactive import when, when_not, clear_flag, set_flag
+from charms.reactive import endpoint_from_flag
+from charmhelpers.core import hookenv
+from charms.layer.kubernetes_common import get_ingress_address
+from charmhelpers.core import unitdata
+
+db = unitdata.kv()
+
+
+@when('ha.connected', 'layer.hacluster.services_configured')
+@when_not('layer.hacluster.configured')
```

Comment:
This should be `layer-hacluster.configured` yeah? That's the name I see used everywhere else in this code.

Reply:
Yep, thanks.